runtime/src/lib.rs

```diff
@@ -639,6 +643,7 @@ pub type Header = generic::Header<BlockNumber, BlakeTwo256>;
 pub type Block = generic::Block<Header, UncheckedExtrinsic>;
 /// The SignedExtension to the basic transaction logic.
 pub type SignedExtra = (
+	frame_system::CheckNonZeroSender<Runtime>,
```

Comment:
Hmmm, this would be a breaking change in the api-client, but It should not be a problem, after we have made the api-client more generic in the course of https://github.com/scs/substrate-api-client/issues/211.

However, I would wait with adapting the SignedExtra until we have resolved the api-client issue. So please revert that.